### PR TITLE
refactor common directives (part of docs effort)

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: modules/@angular/common/src/directives/ng_style.ts [hunk 1]
```diff
@@ -79,34 +75,34 @@ export class NgStyle implements DoCheck {
...
+        (record: KeyValueChangeRecord) => this._setStyle(record.key, record.currentValue));
   }
 
-  private _setStyle(name: string, val: string): void {
-    const nameParts = name.split('.');
-    const nameToSet = nameParts[0];
-    const valToSet = isPresent(val) && nameParts.length === 2 ? `${val}${nameParts[1]}` : val;
+  private _setStyle(nameAndUnit: string, value: string): void {
+    const [name, unit] = nameAndUnit.split('.');
+    value = value !== null && value !== void(0) && unit ? `${value}${unit}` : value;
```
Comment (pkozlowski-opensource) [inline modules/@angular/common/src/directives/ng_style.ts]: Uhh, `value !== null && value !== void(0)` is longer and less readable than `isPresent(value)`.

I understand that we want to remove facades but IMO we've got / had 2 types of functions under the "facades" umbrella: 
* "real" facades that were masking / abstracting differences between TS and Dart => those we need to remove
* helper functions (`isBlank`, `isPresent`) - those are valuable util functions and I would prefer that we keep those (still I understand that this is tricky to share those between modules / barrels but still...)

Comment (pkozlowski-opensource) [inline modules/@angular/common/src/directives/ng_style.ts]: Funny enough you've left `isPresent` in https://github.com/angular/angular/pull/11467 :-)

Comment (vicb) [inline modules/@angular/common/src/directives/ng_style.ts]: Agreed. PR to make them available and I'll use them.
See Jason facade PR for the right place to add them.
Thanks

File: tools/public_api_guard/common/index.d.ts [hunk 2]
```diff
@@ -122,15 +122,15 @@ export declare class NgFor implements DoCheck, OnChanges {
     ngForOf: any;
     ngForTemplate: TemplateRef<NgForRow>;
     ngForTrackBy: TrackByFn;
-    constructor(_viewContainer: ViewContainerRef, _templateRef: TemplateRef<NgForRow>, _iterableDiffers: IterableDiffers, _cdr: ChangeDetectorRef);
+    constructor(_container: ViewContainerRef, _template: TemplateRef<NgForRow>, _differs: IterableDiffers, _cdr: ChangeDetectorRef);
```
Comment (IgorMinar) [inline tools/public_api_guard/common/index.d.ts]: keep as is

File: tools/public_api_guard/common/index.d.ts [hunk 3]
```diff
@@ -122,15 +122,15 @@ export declare class NgFor implements DoCheck, OnChanges {
...
     ngDoCheck(): void;
     ngOnChanges(changes: SimpleChanges): void;
 }
 
 /** @stable */
 export declare class NgIf {
     ngIf: any;
-    constructor(_viewContainer: ViewContainerRef, _templateRef: TemplateRef<Object>);
+    constructor(_container: ViewContainerRef, _template: TemplateRef<Object>);
```
Comment (IgorMinar) [inline tools/public_api_guard/common/index.d.ts]: ditto

**Pull request merged**